lib/qiita/events/qiita_delika_202203/markdown.ex

```diff
@@ -1,36 +1,41 @@
-defmodule Qiita.Qiitadelika202203 do
-  def run do
-    Qiita.Api.patch_item(
-      markdown(),
-      false,
-      [
-        %{"name" => "Elixir"},
-        %{"name" => "delika"},
-        %{"name" => "Qiitadelika"},
-        %{"name" => "40代駆け出しエンジニア"},
-        %{"name" => "AdventCalendar2022"}
-      ],
-      "【毎日自動更新】データに関する記事を書こう！ LGTMランキング！",
-      "b10fa94764aaaa2c6db1"
-    )
+defmodule Qiita.Events.Qiitadelika202203.Markdown do
+  @behaviour Qiita.Events.Markdown
+
+  defmodule Data do
+    defstruct item_count: 0, tables: {}
   end
 
-  def items do
-    start = DateTime.new(~D[2022-03-13], ~T[15:00:00.000], "Etc/UTC") |> elem(1)
-    ending = DateTime.new(~D[2022-04-30], ~T[15:00:00.000], "Etc/UTC") |> elem(1)
+  alias Qiita.Events.Qiitadelika202203.Markdown.Data
+  alias Qiita.Events.TableUtils
 
-    Qiita.Api.items("tag:Qiitadelika OR tag:delika")
-    |> Enum.filter(fn %{"created_at" => created_at} ->
-      Timex.between?(created_at, start, ending, inclusive: :start)
-    end)
+  def generate(items) do
```

Comment:
This is optional but it is nice to explicitly mark a callback function with `@impl`.

```suggestion
  @impl true # or @impl Qiita.Events.Markdown
  def generate(items) do
```